addons/godot_pixelorama_importer/pixelorama_file_open/open.gd

```diff
@@ -0,0 +1,70 @@
+extends EditorInspectorPlugin
+
+var tex:StreamTexture
+var user_config_req:bool = true
+var pixel_path:String
+
+var ctl
+
+func _init():
+	#load config file settings
+	var cfg = ConfigFile.new()
+	var err = cfg.load("user://pixel_plugin.ini")
+	if err == 7:
```

Comment:
Use ERR_FILE_NOT_FOUND, so it's easier to understand what the code does :P

Reply:
Noted.